### DISCUSSION TITLE
"Set services to Manual" breaks amd chipset update

### DISCUSSION THREAD

**Discussion by AK220011** [ChrisTitusTech/winutil#748] [General]
Using this feature breaks amd chipset update as the update says that drivers are only meant for amd systems as if I am not using amd, one of the services must be responsible for detecting or allowing a chipset update/install. Also in the future if windows defender will be able to be disabled, I noticed windows does not work at all properly withought the antimalware service executable even though it's useless if you aren't using windows defender but a 3rd party AV, group policy does not disable it either. For example apps take 20-30 seconds to open up without this service running.

**Comment by Tech-Ezralow** [comment]
Where can i get the list of all the services that are set to manual?

**Reply by vitaliyNet** [reply]
https://christitustech.github.io/winutil/dev/tweaks/Essential-Tweaks/Services/
